spec/controllers/admin/enterprise_fees_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: false
+
+require 'spec_helper'
+require 'open_food_network/order_cycle_permissions'
+
+RSpec.describe Admin::EnterpriseFeesController do
+  before { @request.env['HTTP_REFERER'] = 'http://test.com/' }
```

Comment:
I'm wondering if this is necessary? It may have been copy/pasted from other specs that are testing redirect behaviour, but not relevant here.

Reply:
It's not needed, nor is  `open_food_network/order_cycle_permissions` above. I just remove them